p2p/server.py

```diff
@@ -46,17 +51,27 @@ def __init__(self,
                  server_address: Address,
                  chaindb: AsyncChainDB,
                  bootstrap_nodes: List[str],
-                 network_id: int
+                 network_id: int,
+                 min_peers: int = 5,
```

Comment:
should this rather go into `p2p/constants.py` maybe?

Reply:
I guess the default config will be configurated with `evm.utils.env`? Right now I can add a `DEFAULT_MIN_PEERS`. Referred to [geth](https://github.com/ethereum/go-ethereum/blob/77da203547b83c70d12552a86bb7beee52f6383e/eth/sync.go#L148), maybe set to `5`?
cc @gsalgado